From fc1a659dfa7d0f859f4275cd032a61a3211630cf Mon Sep 17 00:00:00 2001
From: Rudi Heitbaum <rudi@heitbaum.com>
Date: Sun, 4 May 2025 01:09:28 +0000
Subject: [PATCH] allow build with gettext 0.24.1

make sure iconv.m4 is available for autoconf to find it
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index e33d6d3c..022546bb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,6 +4,7 @@
 AC_PREREQ([2.61])
 AC_INIT([MuMuDVB], [m4_join([_], [2.1.0], m4_chomp_all(m4_esyscmd([git rev-parse HEAD])))], [mumudvb@braice.net])
 AC_CONFIG_SRCDIR([src/mumudvb.c])
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE
 AC_CONFIG_HEADERS([src/config.h])
 
@@ -16,6 +17,7 @@ AC_PROG_MAKE_SET
 dnl
 dnl Iconv stuff
 dnl
+AM_GNU_GETTEXT_VERSION([0.15])
 AM_ICONV
 
 dnl
